run-proposal.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+# This script is used on the CI to print proposal output for the highest numbered .sol file in the PR.
```

Comment:
The CI will pass as there will be no `output.json` generated. This can be referenced here https://github.com/moonwell-fi/moonwell-contracts-v2/actions/runs/9155079116/job/25166773990. There are no changes in the proposals and the CI passes. We have the same script here as well.